### PR TITLE
Fixes for local storage and db for enterprise.

### PR DESCRIPTION
**Pull request by jorgeblacio**

### CODE REVIEW

File: src/main/kotlin/com/criptext/mail/db/EventLocalDB.kt [hunk 1]
```diff
@@ -45,9 +45,7 @@ class EventLocalDB(private val db: AppDatabase, private val filesDir: File, priv
     }
 
     fun logoutNukeDB(activeAccount: ActiveAccount) {
-        val username = if(activeAccount.domain == Contact.mainDomain) activeAccount.recipientId
-        else activeAccount.userEmail
-        EmailUtils.deleteEmailsInFileSystem(filesDir, username)
```
Comment (danieltigse) [inline src/main/kotlin/com/criptext/mail/db/EventLocalDB.kt]: it's horrible

File: src/main/kotlin/com/criptext/mail/db/EventLocalDB.kt [hunk 2]
```diff
@@ -237,9 +237,9 @@ class EventLocalDB(private val db: AppDatabase, private val filesDir: File, priv
         }
     }
 
-    fun updateUserName(recipientId: String, name: String, accountId: Long) {
-        db.contactDao().updateContactName("$recipientId@${Contact.mainDomain}", name, accountId)
```
Comment (danieltigse) [inline src/main/kotlin/com/criptext/mail/db/EventLocalDB.kt]: it's horrible

File: src/main/kotlin/com/criptext/mail/db/EventLocalDB.kt (hunk 1 above)
Comment (jorgeblacio) [reply src/main/kotlin/com/criptext/mail/db/EventLocalDB.kt]: utterly horrible

File: src/main/kotlin/com/criptext/mail/db/EventLocalDB.kt (hunk 2 above)
Comment (jorgeblacio) [reply src/main/kotlin/com/criptext/mail/db/EventLocalDB.kt]: utterly horrible

**Pull request merged**